controllers/posts_controller.js

```diff
@@ -0,0 +1,27 @@
+const Post = require("../models/posts_model");
+
+const createPost = async (req, res) => {
+  const postBody = req.body;
+  try {
+    const post = await Post.create(postBody);
+    res.status(201).send(post);
+  } catch (error) {
+    res.status(400).send(error.message);
+  }
+};
+
+const getAllPosts = async (req, res) => {
+  const filter = req.query.sender;
```

Comment:
Why use a filter if the assignment says to get all posts in one JSON without any filters?